nova/bigvm/manager.py

```diff
@@ -60,16 +61,16 @@ def __init__(self, *args, **kwargs):
                                  prepare_empty_host_for_spawning_interval,
                                  run_immediately=True)
     def _prepare_empty_host_for_spawning(self, context):
-        """Handle freeing up hosts per hv_size per az for spawning
+        """Handle freeing up hosts per hv_size per VC for spawning
 
         The general workflow is:
-        1) Find all hypervisor sizes (hv_size) existing in an availability
-           zone (az).
-        2) Choose one host per hv_size per az and create a child resource
+        1) Find all hypervisor sizes (hv_size) existing in a vCenter (VC)
+           existing in an availability zone (az).
```

Comment:
"existing in a vCenter (VC) _within_ an availability zone (az)." maybe?

Reply:
Sure, if that helps someone understand it.